$OpenBSD: patch-source_http_c,v 1.1 2012/03/26 08:09:01 dcoppa Exp $

Stop saying "DEBUG: Chunked!" when encountering chunked http transfers
(upstream git commit 679a0e238d78864c4340cedc3615aa234ef3e440)

--- source/http.c.orig	Mon Mar 26 09:52:55 2012
+++ source/http.c	Mon Mar 26 09:53:10 2012
@@ -157,9 +157,6 @@ char ** fetch(const char * url, FILE ** handle, const 
 		return NULL;
 	}
 
-	if(chunked)
-		puts("DEBUG: Chunked!");
-
 	while(!feof(fd)) {
 		line = NULL;
 		size = 0;
